src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -141,41 +202,69 @@ export class ResultsViewPageStore {
     }, []);
 
     readonly studies = remoteData({
-        invoke: async()=>([await client.getStudyUsingGET({studyId: this.studyId})])
+        invoke: ()=>Promise.all(this.studyIds.map(studyId=>client.getStudyUsingGET({studyId})))
```

Comment:
I guess we don't need async because we already return promise?

Reply:
also, might be nice to return map keyed by studyId?

Reply:
if that need comes up we can definitely derive it